src/components/Forms/CustomInputField.tsx

```diff
@@ -0,0 +1,27 @@
+import { TextField } from "@mui/material";
+
+type Props = {
+  type: string;
+  id: string;
+  name: string;
+  label: string;
+  currentValue: string | number;
+  shrink?: boolean;
+  multiline?: boolean;
+  changeHandler: (event: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+export const CustomInputField = (props: Props) => {
+  return (
+    <TextField
+      type={props.type}
+      multiline={props.multiline}
+      id={props.id}
+      name={props.name}
+      label={props.label}
+      onChange={props.changeHandler}
+      value={props.currentValue}
+      InputLabelProps={{ shrink: props.shrink }}
```

Comment:
A good way you can shorten all of this and not have to write each property and it's corresponding `props` property is using the object spread operator:  
`<TextField {...props} />`

This way all of the props passed to the `CustomInputField` will automatically be passed down instead of having to explicitly define them in the custom component.